src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -177,10 +177,10 @@ def register_options(cls, register):
       default=cls.JvmCompileWorkflowType.zinc_only, metavar='<workflow>',
       help='The default workflow to use to compile JVM targets. This is overriden on a per-target basis with the force-compiler-tag-prefix tag.', fingerprint=True)
 
-    register('--workflow-override', type=cls.JvmCompileWorkflowType,
+    register('--scala-workflow-override', type=cls.JvmCompileWorkflowType,
       choices=cls.JvmCompileWorkflowType.all_values(),
       default=None, metavar='<workflow_override>',
-      help='The workflow to use to compile JVM targets, overriding the "workflow" option as well as any force-compiler-tag-prefix tags applied to targets. An example use case is to quickly turn off outlining workflows in case of errors.', fingerprint=True)
+      help='The workflow to use to compile Scala targets, overriding the "workflow" option as well as any force-compiler-tag-prefix tags applied to targets. An example use case is to quickly turn off outlining workflows in case of errors.', fingerprint=True)
```

Comment:
While we don't have an explicit "experimental" label for flags, I think that we currently know that this has-been/will-be experimental. Should include something to that effect in the help.

Reply:
Good idea, added disclaimer.

Reply:
Thanks. Context I forgot to include (was distracted) is that generally flag renames require a deprecation cycle: labeling this experimental makes it clearer why we're not worrying about it in this case.

Will merge on green.